composer.json

```diff
@@ -1,18 +1,22 @@
 {
-    "name": "google/proto-client-php",
+    "name": "google/proto-client",
```

Comment:
Non-blocking question - the github repo is still called 'proto-client-php', is this something that we should keep consistent?